File: ILT.IHR.APP/ClientApp/src/app/core/providers/data.provider.ts
import { Injectable } from '@angular/core';

@Injectable()
export class DataProvider {
    public storage: any;
    public caseSearchInfo: any;
    public EmployeeFilters: any;
    public country: string
    public status: string
    public employeeType: any[]
    public DefaultPageSize: number;
    public table: any;
    public TabIndex: number;
    public constructor(){
    }
}

File: ILT.IHR.UI/Service/DataProvider.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using BlazorTable;

namespace ILT.IHR.UI.Service
{
    public class DataProvider
    {
        public object storage;
        public string country;
        public string status;
        public List<IMultiSelectDropDownList> employeeType;
        public int DefaultPageSize;
        public object table;
        public int TabIndex;
            
    }
}
